Classes/Configuration/BackendConfigurationManager.php

```diff
@@ -110,7 +110,7 @@ protected function getPageIdFromRecordIdentifiedInEditUrlArgument() {
 		if ('pages' === $table && 'new' === $command) {
 			// if TYPO3 wants to insert a new page, URL argument is already the PID value.
 			return $id;
-		} elseif (0 <> $id) {
+		} elseif (0 <> $id && 'pages' !== $table) {
```

Comment:
I suggest instead in line 127 where `'pid'` is used, to replace the straight usage of `'pid'` with: `'pages' !== $table ? 'pid' : 'uid'`. This would give us the same behavior but would be easier to understand in a year or two ;)